web/beacon-app/src/components/common/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+export interface CheckboxProps {
+  onClick?: () => void;
+  className?: string;
+  id?: string;
+  label: string;
+  dataCy?: string;
+}
+
+function Checkbox({ onClick, className, id, label, dataCy }: CheckboxProps) {
```

Comment:
While attempting to resolve the bug with the `Checkbox` component from the design system, I noticed that the label/text wouldn't sit next to the checkbox. Part of the text would appear above the checkbox. 

After not being able to resolve that issue in the design system, I thought it would prove easier to create a reusable `Checkbox` component in Beacon to fix the bug while ensuring the text continues to appear in line next to the checkbox.